MAUI/ProgressBar/migration.md

```diff
@@ -0,0 +1,587 @@
+title: Migrate from Xamarin to .NET MAUI  ProgressBar | Syncfusion
+description: Learn here all about Migrating from Syncfusion Xamarin SfProgressBar to Syncfusion .NET MAUI SfProgressBar control and more.
+platform: MAUI
+control: ProgressBar
+documentation: ug
+---  
+
+# Migrate from Xamarin.Forms SfProgressBar to .NET MAUI SfProgressBar
+
+To make the migration from the [Xamarin SfProgressBar](https://www.syncfusion.com/xamarin-ui-controls/xamarin-progressbar) to `.NET MAUI SfProgressBar` easier, most of the APIs from the Xamarin SfProgressBar were kept in the.NET MAUI SfProgressBar. However, to maintain the consistency of API naming in the .NET MAUI SfProgressBar, some of the APIs have been renamed. Please find the difference in the following topics.
+
+## Initialize control
+
+To initialize the control, import the ProgressBar namespace and initialize SfProgressBar as shown in the following code sample.
+
+<table>
+<tr>
+<th>Xamarin SfProgressBar</th>
+<th>.NET MAUI SfProgressBar</th></tr>
+<tr>
+<td>
+{% tabs %}
+
+{% highlight xaml %}
+
+<ContentPage
+    xmlns:progressBar="clr-namespace:Syncfusion.XForms.ProgressBar;assembly=Syncfusion.SfProgressBar.XForms">
+
+    ...     
+    <progressBar:SfLinearProgressBar />
+
+    <progressBar:SfCircularProgressBar />
+    ...
+</ContentPage>
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+using Syncfusion.XForms.ProgressBar;
+
+...
+
+SfLinearProgressBar linearProgressBar = new SfLinearProgressBar();
+
+SfCircularProgressBar circularProgressBar = new SfCircularProgressBar();
+
+...
+
+{% endhighlight %}
+
+{% endtabs %}
+
+</td>
+<td>
+{% tabs %} 
+
+{% highlight xaml %}
+
+<ContentPage
+    xmlns:progressBar="clr-namespace:Syncfusion.Maui.ProgressBar;assembly=Syncfusion.Maui.ProgressBar">
+
+    ...
+    <progressBar:SfLinearProgressBar />
+
+    <progressBar:SfCircularProgressBar />
+    ...
+
+</ContentPage>
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+using Syncfusion.Maui.ProgressBar;
+...
+
+SfLinearProgressBar linearProgressBar = new SfLinearProgressBar();
+
+SfCircularProgressBar circularProgressBar = new SfCircularProgressBar();
+
+...
+
+{% endhighlight %}
+
+{% endtabs %}
+</td></tr>
+</table>
+
+## Common Properties
+
+<table> 
+<tr>
+<th>Xamarin SfProgressBar</th>
+<th>.NET MAUI SfProgressBar</th>
+<th>Description</th></tr>
+<tr>
+<td>{{'[EasingEffect](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.ProgressBar.ProgressBarBase.html#Syncfusion_XForms_ProgressBar_ProgressBarBase_EasingEffect)'| markdownify }}</td>
+<td>{{'AnimationEasing'| markdownify }}</td>
+<td>Gets or sets a value that specifies the easing effect for progress animation.</td></tr>
+<tr>
+<td>{{'[GapWidth](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.ProgressBar.ProgressBarBase.html#Syncfusion_XForms_ProgressBar_ProgressBarBase_GapWidth)'| markdownify }}</td>
+<td>{{'SegmentGapWidth'| markdownify }}</td>
+<td>Gets or sets the value that determine the gap between the segments.</td></tr>
```

Comment:
<td>Gets or sets the value that determines the gap between the segments.</td></tr>

Reply:
done